lib/iris/tests/results/analysis/exponentiate.cml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" ?>
 <cubes xmlns="urn:x-iris:cubeml-0.2">
-  <cube core-dtype="float64" dtype="float64" fill_value="-1e+30" units="kelvin^4">
```

Comment:
The test in question changes the cube data `dtype` from `float32` to `float64`, hence the change in `fill_value` representation ...